redisinsight/api/src/modules/rdi/client/api.rdi.client.ts

```diff
@@ -62,7 +62,8 @@ export class ApiRdiClient extends RdiClient {
         jobs,
       };
     } catch (e) {
-      throw wrapRdiPipelineError(e);
+      const message: string = e.response?.data?.detail || e.message || '';
```

Comment:
I think instead of repeat this code several times better move this logic inside `wrapRdiPipelineError`

Reply:
and the last value should be `undefined` for all default values messages inside the `RdiPipeline.....Exception(.....)`